### ISSUE TITLE
add backpack to various ethereum wallet connectors

### ISSUE THREAD

**Issue by tomlinton** [coral-xyz/backpack#767]
Once page injected provider is complete.

- wagmi - https://github.com/wagmi-dev/wagmi/blob/main/packages/core/src/utils/getInjectedName.ts
- web3-onboard - https://github.com/blocknative/web3-onboard/blob/v2-web3-onboard-develop/packages/injected/src/wallets.ts

**Comment by armaniferrante** [comment]
Another one used by magic eden https://github.com/Uniswap/web3-react

**Comment by armaniferrante** [comment]
Is this still blocked?

**Comment by tomlinton** [comment]
No, although we may want to alter what we expose on window depending on how the integrations look

**Comment by armaniferrante** [comment]
Sounds good. Would be good to get this in so that we can start the grind of asking ethereum teams to integrate Backpack.

**Comment by tomlinton** [comment]
Closing in favour of #1181, #1182, #1183 to make it easier for external contributors to take one of these on
